cantabular-import/minio/IMPORTANT.md

```diff
@@ -0,0 +1,8 @@
+### IMPORTANT! ###
+
+## Do not delete the `data` directory! This directory has special permissions ##
```

Comment:
Where is the data directory? I'm guessing that this data directory gets created inside the minio docker container? My understanding of volumes is slightly lacking but the deps.yml shows two things named 'data' under volumes (./minio/data:/data). It would be good to be more specific in this comment I think.

Reply:
Hmm, I thought the empty directory should have been checked in.. maybe because I've added the entire contents to the .gitignore it hasn't.. it needs to though, will investigate

Reply:
Ok have added a .gitignore in the directory itself so it maintains itself. Hopefully it's obvious now as it's in the same root as the IMPORTANT.md?

Reply:
The readme looks good now - and I can see the data directory there now too.